### DISCUSSION TITLE
Question about roadmap

### DISCUSSION THREAD

**Discussion by Rmarian** [etcd-io/jetcd#1250] [General]
Hi @lburgazzoli ,

We want to migrate from etcd v2 to v3 on our project and would like to use jetcd java client going forward. To this end we would have a question:
I saw that there is yet no version 1.0 released, latest that I see being 0.7. Does this mean the client is still in beta and might be unstable for production use? If yes is there a roadmap and a timeline for 1.0 release? 

Many thanks,
Radu.

**Comment by lburgazzoli** [comment]
No, even if jetcd is still version 0.7, it does not mean that it is in beta and as far as I know, there are a number of projects already using it. 

FWIW, I don't have any eta to release a v1.0 at this stage to be honest.
